ocaml/idl/datamodel_pool.ml

```diff
@@ -1331,6 +1331,9 @@ let t =
             "coordinator_bias"
             "true if bias against pool master when scheduling vms is enabled, \
              false otherwise"
+        ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:DateTime
+            ~default_value:(Some (VDateTime (Date.of_float 0.)))
```

Comment:
This should use `Date.never`

Reply:
`never` is deprecated, `Date.epoch` should be used instead